README.md

```diff
@@ -0,0 +1,109 @@
+<a id="top"></a>
+
+# peAk

```

Comment:
Tiny, nitpicky feedback: it would have been nice to change the favicon of your application away from the generic Spring leaf.